2023/incoming/trimming-audio-files-with-audio-nama.pod

```diff
@@ -0,0 +1,338 @@
+
+	 No. Name       Requested  Status  Source                Destination   Vol   Pan
+	===============================================================================
+	  1  Main                  MON     Main bus              CH 1/2        100    50
+	  2  Mixdown               OFF     track Main            --             --    --
+	  3  carol                 MON     carol sequence        Main bus      100    50
+	  4  carol-1-carol-v       PLAY    carol_1.wav           carol sequen  100    50
+	  5  carol-2-carol-v       PLAY    carol_1.wav           carol sequen  100    50
+	  6  carol-3-carol-v       PLAY    carol_1.wav           carol sequen  100    50
+	  7  carol-4-carol-v       PLAY    carol_1.wav           carol sequen  100    50
+
+In the chain setup below we see that C<select> and C<playat> terms
+are used in the inputs to define a region of the audio file and play it 
+with a given delay. The values were generated from our list
+of marks.
+
+	nama christmas carol > chains
+
+	# audio inputs
+
+	-a:1 -f:s16_le,16,44100,i -i:loop,Main_in
+	-a:3 -f:s16_le,16,44100,i -i:loop,carol_in
+	-a:4 -f:s16_le,2,44100 -i:select,40,300,/home/jroth/nama/christmas/.wav/carol_1.wav
+	-a:5 -f:s16_le,2,44100 -i:playat,300,select,380,300,/home/jroth/nama/christmas/.wav/carol_1.wav
+	-a:6 -f:s16_le,2,44100 -i:playat,600,select,720,300,/home/jroth/nama/christmas/.wav/carol_1.wav
+	-a:7 -f:s16_le,2,44100 -i:playat,900,select,1060,300,/home/jroth/nama/christmas/.wav/carol_1.wav
+
+	# audio outputs
+
+	-a:1 -o:alsa,default
+	-a:3 -f:s16_le,16,44100,i -o:loop,Main_in
+	-a:4,5,6,7 -f:s16_le,16,44100,i -o:loop,carol_in
+
+The mixdown configuration is the same, with the output of chain 1 directed to a file. 
+
+=head2 Thanks for reading!
+
+There are many other things you can do with Nama but that's all for today!
+
+=head2 Showcase
+
+Jeanette Claassen has produced a large number of
+L<original|https://www.youtube.com/@jeanette_c3896/videos>
+L<songs|https://juliencoder.de/nama> using Nama in
+combination with various other music software and hardware.
+She began recording music using Ecasound and migrated to
+Nama as it became sufficiently featureful.
+
+=head2 Notable dependencies
+
+ L<Git::Repository>: Manage project history, branching, undo
+ L<Log::Log4perl>: Logging, debugging
+ L<Parse::RecDescent>: Command language parser
+ L<Object::Tiny>: Accessor generator
+ L<Role::Tiny>: Breaking up large classes
+ L<Term::ReadLine::Gnu>: Terminal support, tab completion, history
+ L<Term::TermKey>: Hotkey mappings
+ L<Graph>: Representing, traversing and transforming the signal processing network
+ L<AnyEvent>: Event loop
+ L<Tk>: GUI, available with the -g option, use at your own risk!
+ L<Data::Section::Simple>: Store and access sections within the __DATA__ block
+ L<Data::Dumper::Concise>, tho I would now recommend L<Data::Printer>.
+ L<Memoize>: Turn expensive, redundant calculations into cheap lookups
+ L<Devel::NYTProf>: Profiling to find and solve bottlenecks
```

Comment:
This appears to be a list. Perhaps it should start with an =over, then be a sequence of =item nodes and finally a =back?